### PR TITLE
xca: update to 2.0.1

### PR DESCRIPTION
**Pull request by dutsky**

### CODE REVIEW

File: srcpkgs/xca/template [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
+revision=1
 build_style=gnu-configure
-make_build_target="all"
-hostmakedepends="pkg-config qt5-tools qt5-host-tools"
+hostmakedepends="pkg-config qt5-tools qt5-host-tools perl"
 makedepends="qt5-devel libressl-devel libltdl-devel"
+depends="qt5-plugin-sqlite"
 short_desc="X Certificate and Key Management"
 maintainer="Denis Revin <denis.revin@gmail.com>"
 license="BSD"
```
Comment (maxice8) [inline srcpkgs/xca/template]: please use SPDX identifier

Comment (dutsky) [reply srcpkgs/xca/template]: as you wish, but Manual.md says: 
```license: A string matching any license file available in /usr/share/licenses. Multiple licenses should be separated by commas, i.e GPL-3, LGPL-2.1.```
maybe it must be changed if you want SPDX there?

Comment (maxice8) [reply srcpkgs/xca/template]: working on it.  https://github.com/voidlinux/void-packages/issues/12219

**Comment by cr6git** [comment]
@dutsky something wrong with the patch?
````
=> xca-2.0.1_1: running post-extract hook: 00-patches ...
=> xca-2.0.1_1: patching: libressl.patch.
1 out of 1 hunk FAILED -- saving rejects to file lib/pki_temp.cpp.rej
````

**Comment by dutsky** [comment]
I just wonder why it is here?! It must not be here :blush:

**Comment by cr6git** [comment]
@dutsky sorry about the patch, it was my fault... :roll_eyes:

**Pull request merged**